### DISCUSSION TITLE
why not using period in the end

### DISCUSSION THREAD

**Discussion by haolu-luobu** [olsh/resharper-structured-logging#28] [Q&A] [answered]
Is there any performance or functional reason to stop using a period in the log?

**Comment by olsh** [comment]
Hi @haolu-luobu 
You can find all analyzers description here https://github.com/olsh/resharper-structured-logging#analyzers
Here is the description for the mentioned one https://github.com/olsh/resharper-structured-logging/blob/master/rules/LogMessageIsSentenceProblem.md

**Comment by haolu-luobu** [comment]
Yes, I find those
`Log event messages are fragments, not sentences; for consistency with other libraries that use Serilog, avoid a trailing period/full stop when possible.`

So avoid trailing period to my understanding, more like a consistency, there won't be any performance or function wise reason.

**Comment by olsh** [comment, marked as answer]
> So avoid trailing period to my understanding, more like a consistency, there won't be any performance or function wise reason.

Correct.
